handbook/engineering/incident_dotcom_deleted_entirely.md

```diff
@@ -0,0 +1,62 @@
+# How to resolve a "Sourcegraph.com is deleted entirely" incident
```

Comment:
True or false: ideally the content in the page would be "run command XYZ to ensure that sourcegraph.com is deployed correctly" (totally fine if the script is interactive and the dev has to make choices).

If true, can we file an issue to capture that work and then link to that issue from the top of this handbook page?

Reply:
True and false.

In a perfect world, yes, that would be ideal. In the real world, I think it is always likely to require at least some of the following:

- A few different commands
- Inspecting the current state in some way, and running different commands based on what you see.
- Having your `gcloud`, `kubectl`, `terraform` commands installed and set up to point at the right projects, etc
- Explaining to someone who is not familiar with our tooling, repository structure, infrastructure, etc. what these magical commands are doing in a critical time without saying "here are 5 page-long readings, go follow those first"

Thus, I imagine this document getting easier and more clear over time -- but likely not ever going away entirely. https://github.com/sourcegraph/sourcegraph/issues/10455 is one such example of how this document will become easier but not go away.